src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/KildaConfigurationService.java

```diff
@@ -51,6 +51,10 @@ public KildaConfiguration getKildaConfiguration() {
      */
     public KildaConfiguration updateKildaConfiguration(KildaConfiguration kildaConfiguration) {
         log.info("Process kilda config update - config: {}", kildaConfiguration);
+        if (Boolean.FALSE.equals(kildaConfiguration.getUseMultiTable())) {
+            throw new IllegalArgumentException("Single table mode was deprecated. OpenKilda doesn't support it "
+                    + "anymore. Configuration property `use_multi_table` can't has 'false' value.");
```

Comment:
Mistake - `can't have`. Or, even better, `can't be 'false'`.

Reply:
fixed